packages/backend/server/src/plugins/gcloud/logging/service.ts

```diff
@@ -1,22 +1,28 @@
 import { LoggingWinston } from '@google-cloud/logging-winston';
 import { LoggerService, Provider } from '@nestjs/common';
-import { createLogger, transports } from 'winston';
+import { createLogger, format, transports } from 'winston';
 
 import { AFFiNELogger as LoggerProvide } from '../../../base/logger';
 import { AFFiNELogger } from './logger';
 
+const moreMetadata = format(info => {
+  info.requestId = LoggerProvide.getRequestId();
+  return info;
+});
+
 export const loggerProvider: Provider<LoggerService> = {
   provide: LoggerProvide,
   useFactory: () => {
     const loggingWinston = new LoggingWinston();
     // Create a Winston logger that streams to Cloud Logging
     const instance = createLogger({
-      level: 'log',
+      level: 'info',
```

Comment:
Not quite sure why it was set to log here before, the actual log is not effective, it should be set to info or verbose level. @Brooooooklyn

Reply:
level: info > log > verbose
verbosity: info < log < verbose

Reply:
There is no `info` level in Nestjs `.d.ts`

![CleanShot 2025-01-20 at 16.21.16@2x.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/Qu1BZHWlS6JUIwBDqAAj/4d0d67ae-53ae-4984-a2bf-35879bff4225.png)

Reply:
The `createLogger` here is provided by `winston`, not `nest`. Check winston's documentation, is there no definition for level `log`? https://github.com/winstonjs/winston?tab=readme-ov-file#logging-levels

Reply:
I was confused by this 2 libraries, let's change it back to `info`

Reply:
I found the key code. In nest-winston, the default level of nest `logger.log()` will be output at winston's `info` level. So it is reasonable for us to change the winston level to `info`. 

https://github.com/gremo/nest-winston/blob/main/src/winston.classes.ts#L23